legacy/builder/print_used_and_not_used_libraries.go

```diff
@@ -29,32 +28,31 @@ type PrintUsedAndNotUsedLibraries struct {
 }
 
 func (s *PrintUsedAndNotUsedLibraries) Run(ctx *types.Context) error {
-	var logLevel string
-	// Print this message as warning when the sketch didn't compile,
-	// as info when we're verbose and not all otherwise
-	if s.SketchError {
-		logLevel = constants.LOG_LEVEL_WARN
-	} else if ctx.Verbose {
-		logLevel = constants.LOG_LEVEL_INFO
-	} else {
+	// Print this message:
+	// - as warning, when the sketch didn't compile
+	// - as info, when verbose is on
+	// - otherwise, output nothing
+	if !s.SketchError && !ctx.Verbose {
 		return nil
 	}
 
-	logger := ctx.GetLogger()
-	libraryResolutionResults := ctx.LibrariesResolutionResults
-
-	for header, libResResult := range libraryResolutionResults {
+	res := ""
+	for header, libResResult := range ctx.LibrariesResolutionResults {
 		if len(libResResult.NotUsedLibraries) == 0 {
 			continue
 		}
-		logger.Fprintln(os.Stdout, logLevel, tr("Multiple libraries were found for \"{0}\""), header)
-		logger.Fprintln(os.Stdout, logLevel, " "+tr("Used: {0}"), libResResult.Library.InstallDir)
+		res += fmt.Sprintln(tr(`Multiple libraries were found for "%[1]s"`, header))
+		res += fmt.Sprintln("  " + tr("Used: %[1]s", libResResult.Library.InstallDir))
```

Comment:
```suggestion
		res += fmt.Sprint("  " + tr("Used: %[1]s", libResResult.Library.InstallDir))
```
See the associated suggestion below for the explanation (GitHub doesn't allow suggestions to cross a - diff so I have to do it in two).

This suggestion should only be accepted if the following one is.